### PR TITLE
Include docker and kubernetes in generated versions

### PR DESCRIPTION
**Pull request by areed**
Also prefer order defined for add-on versions on the Installer class since
that determines latest. We do not want weave 2.7.0 above 2.6.5, for
example, since 2.6.5 is latest and 2.7.0 has bugs.

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "nyc": "^11.4.1",
     "snyk": "^1.41.1",
     "start-server-and-test": "^1.9.0",
-    "ts-node": "^4.0.2",
+		"ts-node": "^4.0.2",
```
Comment (salahalsaleh) [inline web/package.json:48]: indentation

Comment (areed) [reply web/package.json]: fixed

File: generate-addons.js [hunk 2]
```diff
@@ -68,11 +69,20 @@ fs.readdir(specDir, (err, files) => {
...
+      if (preferredVersions[file]) {
+        sv = preferredVersions[file];
+      }
       sv.unshift("latest");
       supportedVersions[file] = sv
     }
   });
 
+  supportedVersions.kubernetes = preferredVersions.kubernetes;
+  preferredVersions.kubernetes.unshift("latest");
```
Comment (salahalsaleh) [inline generate-addons.js:81]: i don't get why doing `unshift` at this point, `preferredVersions.kubernetes` is not used after this line. unless you meant to use `supportedVersions`?

Comment (salahalsaleh) [reply generate-addons.js:81]: i guess modifying `preferredVersions` would also modify `supportedVersions`, but it's less clear this way

Comment (areed) [reply generate-addons.js:81]: I'll change to `supportedVersions`

**Review by salahalsaleh — APPROVED** [review]

**Review by salahalsaleh — APPROVED** [review]

**Pull request merged**